nixos/modules/installer/sd-card/sd-image.nix

```diff
@@ -170,6 +184,7 @@ in
 
     sdImage.storePaths = [ config.system.build.toplevel ];
 
+    system.build.firmware = config.sdImage.populateFirmware;
```

Comment:
```suggestion
```

We shouldn't make this any form of "API" any more than it is. Also `system.build` is kind of a weird bag of stuff, where we should carefully consider whether or not we need to add things into.

Reply:
I agree, dropped.